scripts/processors/cams_processor.py

```
@@ -0,0 +1,343 @@
+            
+            # Add H3 index for spatial aggregation
+            if not result.empty:
+                result['h3_index_res8'] = result.apply(
+                    lambda row: h3.latlng_to_cell(row['latitude'], row['longitude'], 8),
+                    axis=1
+                )
+                
+            logger.info(f"  Extracted {len(result)} records from {nc_path.name}")
+            return result
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)